### PR TITLE
Capstone African Conference on Education Website

### PR DESCRIPTION
**Pull request by YKerroum**
# Create a website for the ACE 2022
## Create a Home page (for mobiles and desktops): 

- Followed the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as requested.
- Added a responsive navbar that, when you click on his menu, takes you to the about page.
- Create a dynamic `featured speakers` section generated from the speakers' Array in JS.
- Add a `More button` to the featured speakers section.

## Create a video to describe the website features (You can find it in the README file also):
[Features video](https://www.loom.com/share/636f8a029734430d9b5bc0aad962eade)

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @YKerroum , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] Kindly add a video presenting the features of the project, as instructed in the project requirements
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                    <input class="checkbox" type="checkbox"/>
+                    <div class="hamburger-lines">
+                        <span class="line line1"></span>
+                        <span class="line line2"></span>
+                        <span class="line line3"></span>
+                    </div>
+                    <div class="logo">
+                        <a href="index.html"><img src="assets/images/conference-logo.png" alt="Conference Logo"></a>
+                    </div>
+                    <ul class="menu-items">
```
[the remark this answers is not in the document]
Comment (YKerroum) [reply index.html:48]: I added a menu in about page to return to the main page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                    <li><img src="assets/images/partner1.png" alt=""></li>
+                    <li><img src="assets/images/partner2.png" alt=""></li>
+                    <li><img src="assets/images/google.png" alt=""></li>
+                    <li><img src="assets/images/mozilla.png" alt=""></li>
+                    <li><img src="assets/images/lumen.png" alt=""></li>
+                </ul>
+            </div>
+        </section>
+        <footer>
+        <div id="footer-container" class="text-normal">
```
[the remark this answers is not in the document]
Comment (YKerroum) [reply index.html:168]: Solved thank you.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                    </ul>
+                </div>
+                <div class="compaign">
+                    <button type="button">ACE 2022 Compaign</button>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section id="intro-container">
```
[the remark this answers is not in the document]
Comment (YKerroum) [reply about.html:64]: Solved

File: index.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+                <div id="featured-speakers-list">
+                        
+                </div>
+            </div>
+        </section>
+        <section id="partners">
+            <div id="partners-container" class="text-normal">
+                <h2>Partner</h2>
+                <ul id="partners-list">
+                    <li><img src="assets/images/partner1.png" alt=""></li>
```
[the remark this answers is not in the document]
Comment (YKerroum) [reply index.html:159]: Solved.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+                    <li><img src="assets/images/mozilla.png" alt=""></li>
+                    <li><img src="assets/images/lumen.png" alt=""></li>
+                </ul>
+            </div>
+        </section>
+        <footer>
+        <div id="footer-container" class="text-normal">
+            <div id="logo">
+                <a href="index.html">
+                    <img src="/assets/images/conference-logo.png" alt="Conference logo">
```
[the remark this answers is not in the document]
Comment (YKerroum) [reply index.html:171]: Solved

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @YKerroum , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Fixed all previous reviewer suggestions ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**